src/test/java/teammates/test/cases/common/StringHelperTest.java

```diff
@@ -208,7 +208,18 @@ public void testRecoverFromSanitizedText(){
         str = Sanitizer.sanitizeForHtml("<text><div> 'param' &&& \\//\\");
         assertEquals("<text><div> 'param' &&& \\//\\",StringHelper.recoverFromSanitizedText(str));
     }
-    
+
+    @Test
+    public void testCountingEmptyStrings() {
```

Comment:
I think the current naming convention is to add `test` to the name of the method under test.